src/core/operations_problem.jl

```diff
@@ -145,7 +147,7 @@ function OperationsProblem{M}(
         export_pwl_vars = export_pwl_vars,
         allow_fails = allow_fails,
         PTDF = PTDF,
-        optimizer_log_print = optimizer_log_print,
```

Comment:
we should keep `optimizer_log` here since this is actually to display in the screen the optimizer iteration log.